### PR TITLE
<langchain>: tracing_enabled incompatibility issue between core 0.1.5 and langchain v0.0.353

### PR DESCRIPTION
**Pull request by xkaiDu**
https://github.com/langchain-ai/langchain/issues/15526

TLDR:

In commit [7eec8f2](https://github.com/langchain-ai/langchain/commit/7eec8f24876fd624573c09ff82270362031e6a5f#) the function [tracing_enabled](https://github.com/langchain-ai/langchain/commit/7eec8f24876fd624573c09ff82270362031e6a5f#diff-aa3d4ef71b404ecd0d1198ccf778a3ccc605d9b6e043057d0db3bd8f7d9c483dL46) was deleted.

the function deletion was included in langchain-core [0.1.5](https://github.com/langchain-ai/langchain/pull/15480), and is being referred to by v0.0.353, which didn't clean up the dependencies yet, not until v0.0.354

Causing the retrievers API of v0.0.353 and some earlier versions to raise ImportError after Jan 3rd, when langchain-core 0.1.5 was released.

### CODE REVIEW

**Comment by hwchase17** [comment]
sorry about this, should be fixed now and we yanked 0.1.5
